webapp/pages/post/_id/_slug/index.vue

```diff
@@ -64,22 +64,41 @@
                 <base-icon name="map-marker" data-test="map-marker" />
                 <span v-if="post.eventVenue">{{ post.eventVenue }}</span>
                 <span v-if="!post.eventIsOnline">
-                  <span v-if="post.eventVenue">-</span>
+                  <span v-if="post.eventVenue">&mdash;</span>
                   {{ post.eventLocationName }}
                 </span>
                 <span v-else>
-                  <span v-if="post.eventVenue">-</span>
+                  <span v-if="post.eventVenue">&mdash;</span>
                   {{ $t('post.viewEvent.eventIsOnline') }}
                 </span>
```

Comment:
Hint: Because row 67 and 71 are the same we could move it as one line before the first `span`.

Reply:
I solved this in my other PR